test/gie/builtins.gie

```diff
@@ -2776,6 +2776,35 @@ operation +proj=isea   +mode=hex +resolution=31
 accept  0 0
 expect  failure
 
+-------------------------------------------------------------------------------
+operation +proj=isea +R=6371007.18091875
+-------------------------------------------------------------------------------
+tolerance 2 mm
+direction inverse
```

Comment:
it would be more convincing if we had the corresponding test for the forward test, to make it obvious that round tripping works.

The way to do it is to remove "direction inverse", switch the expect and accept parameters, and add a "roundtrip 1" line after each expect. So something like

accept -168.75 58.282525588539
expect -19186144.872455853968859 3323137.771243878174573
roundtrip 1

accept 11.25 58.282525588539
expect -15348915.898387057706714 9969413.314565381035209
roundtirp 1

...

Reply:
We can do that. The expect values come directly from copy-pasting results of the forward `+proj=isea` so I am convinced myself that it does work ;)

Reply:
I've turned on roundtrip for all test cases, including the previous test cases using `+a=6400000`.
They now all pass (after sharing more of the constants between the forward and inverse, which I hope generaly improves the accuracy) but I had to update values and bump up tolerance to 1mm for everything.